pyfar/classes/audio.py

```diff
@@ -1249,3 +1249,92 @@ def _divide(a, b):
 
 def _power(a, b):
     return a**b
+
+
+def match_fft_norm(fft_norm_1, fft_norm_2, division=False):
```

Comment:
Add `_` to clearly mark it as a private function
```suggestion
def _match_fft_norm(fft_norm_1, fft_norm_2, division=False):
```

Reply:
check.